modules/quotas-and-limits-ibm-power-vs.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc
+
+:_content-type: CONCEPT
+[id="quotas-and-limits-ibm-power-vs_{context}"]
+= Quotas and limits on {ibmpowerProductName} Virtual Server
+
+The {product-title} cluster uses a number of IBM Cloud and {ibmpowerProductName} Virtual Server components, and the default quotas and limits affect your ability to install {product-title} clusters. If you use certain cluster configurations, deploy your cluster in certain regions, or run multiple clusters from your account, you might need to request additional resources for your IBM Cloud VPC account.
+
+For a comprehensive list of the default IBM Cloud VPC quotas and service limits, see the IBM Cloud documentation for link:https://cloud.ibm.com/docs/vpc?topic=vpc-quotas[Quotas and service limits].
+
+[discrete]
+== Virtual Private Cloud (VPC)
+
+Each {product-title} cluster creates its own VPC. The default quota of VPCs per region is 10. If you have 10 VPCs created, you will need to increase your quota before attempting an installation.
+
+[discrete]
+== Application load balancer
+
+By default, each cluster creates two application load balancers (ALBs):
+
+* Internal load balancer for the master API server
```

Comment:
@clnperez @mjturek looking for exact term to be used here. Could you please suggest ? thanks.

Reply:
According to the doc link, we should use `control plane` instead of `master`. For example, for this the lb heading, let's use `Internal load balancer for the control plane API server`

Reply:
Ack.